src/Controller/Services.php

```diff
@@ -73,6 +73,14 @@ public function processRequest(Request $request, RouteMatchInterface $route_matc
     }
     // Get the data from the plugin.
     $data = $service_def->processRequest($request, $route_match, $this->serializer);
+    $code = $service_def->getPluginDefinition()['response_code'];
+    $headers = [];
+    $messages = drupal_get_messages();
```

Comment:
Very cool, so developers just need to call drupal_set_message during their code, like the rest of the development?